Libraries/plist/Sources/UnixTime.cpp

```diff
@@ -16,8 +16,13 @@ using plist::UnixTime;
 void UnixTime::
 Decode(uint64_t in, struct tm &out)
 {
+#if _WIN32
+    time_t const t = in;
```

Comment:
Needed to get it to compile, not sure why. I don't see `gmtime_s` in `man gmtime` on macOS, not sure about Linux. I think it's from a too-new version of C or POSIX or something? Maybe there's a C++ stdlib API for this I could use instead?